File: Entidad/Persona.cs
using System;

namespace Entidad
{
    public class Persona
    {
        public string Identificacion{get;set;}
        public string Nombre { get; set; }
        public int Edad { get; set; }
        public string Sexo { get; set; }
        public decimal Pulsacion { get; set; }

        public void CalcularPulsacion()
        {
            if (Sexo.Equals("F") || Sexo.Equals("f"))
            {
                Pulsacion = (220 - Edad )/10;
            }else{
                 Pulsacion = (210 - Edad )/10;

            }
        }

    }
}

File: pulsacionesdotnet/ClientApp/src/app/Pulsacion/moduls/persona.ts
export class Persona {
identificacion: string;
nombre: string;
sexo: string;
edad: number;
pulsacion: number;
CalcularPulsacion() {
    if (this.sexo === 'm') {
        this.pulsacion = ( 210 - this.edad) / 10;
    } else if (this.sexo === 'f') {
        this.pulsacion = ( 220 - this.edad) / 10;
    }
}
}
